src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/__init__.py

```diff
@@ -179,11 +180,28 @@ def _handle_env_configuration():
         # no env exists so create first environment
         _create_or_update_env('default')
 
+def _handle_telemetry_configuration():
```

Comment:
Instead of creating a new method, this should go under the `_handle_global_configuration` method.
Right after the default output type prompt, we should have the telemetry prompt here.
Then, you don't need to have all this global_config logic.